presto-main/src/main/java/com/facebook/presto/sql/analyzer/MaterializedViewPlanValidator.java

```diff
@@ -120,11 +159,13 @@ protected Void visitComparisonExpression(ComparisonExpression node, Materialized
     {
         private boolean isProcessingJoinNode;
```

Comment:
If you'd like, would you be able to help rename this flag as isWithinJoinCriteria, to be consistent?

Reply:
Good idea! Dont like the current name